### DISCUSSION TITLE
Python sdk for pocketbase?

### DISCUSSION THREAD

**Discussion by jet10000** [pocketbase/pocketbase#502] [Q&A]
I want to use with flet.dev

https://flet.dev/

**Comment by ganigeorgiev** [comment]
There is no official Python SDK for PocketBase, but you can always use the REST API:

- https://pocketbase.io/docs/api-records/
- https://pocketbase.io/docs/api-users/

**Comment by dsm-72** [comment]
@ganigeorgiev @jet10000  there is [this](https://github.com/vaphes/pocketbase) but it is outdated for current api.
It would be nice if there was an official python SDK or if there was at least an example

**Comment by romayengineer** [comment]
There is no official SDK for Python but this repository may be what you need, check it out here https://github.com/vaphes/pocketbase
